lib/can/include/CANMsg.h

```diff
@@ -69,6 +70,14 @@ class CANMsg : public CANMessage {
     len                    = sizeof(T);
   }
 
+  /** Set the payload data with custom length
+   */
+  template <class T>
+  void setPayload(const T value, size_t length) {
+    setPayload(value);
+    len = length;
+  }
```

Comment:
Added this function to be able to specify the length of the CAN payload instead of using `len = sizeof(T)`, since I'm always passing in `uint8_t[8]`'s